File: in_Python/158A_Next_Round.py
participants,target = map(int,input().split())
winners = 0

Scores = [];
Scores = list(map(int,input().split()))

for i in range(participants):
    if Scores[i] > 0 and Scores[i] >= Scores[target-1]:
        winners+=1
print(winners)

File: in_C/158A_Next_round.c
#include<stdio.h>

int main() {
    int participants,target;
    scanf("%d%d",&participants,&target);

    int winners = 0;
    int scores[participants];

    for(int i = 0;i < participants;i++) {
        scanf("%d",&scores[i]);
    }
    for (int j = 0;j < participants;j++) {
        if (scores[j] > 0 && scores[j] >= scores[target-1]) {
            winners++;
        }
    }
    printf("%d",winners);
}